prometheus/histogram.go

```diff
@@ -308,23 +308,23 @@ func NewHistogramVec(opts HistogramOpts, labelNames []string) *HistogramVec {
 }
 
 // GetMetricWithLabelValues replaces the method of the same name in
-// MetricVec. The difference is that this method returns a Histogram and not a
+// MetricVec. The difference is that this method returns an Observer and not a
 // Metric so that no type conversion is required.
```

Comment:
The Write call will only happen in test code (and arguably you should collect real Metrics through the Vec Collector instead of testing a single metric coming out of a Vec). In fact, the Metric interface will disappear pretty much in v0.10, but that's a different story.

In any case, referring to the Write method here would probably be more confusing than enlightening.